Makefile

```diff
@@ -35,7 +36,7 @@
 
 .PHONY: all
 
-# Mocks
+# ------------------------------------------------------- MOCKS -------------------------------------------------------------
```

Comment:
after we ship illuminate and v3 @sourabhmarathe we'll sit down and rework these growing makefiles
things can be made less append-for-the-win by using some variables and some better composition....

this works til then